includes/rest-api.php

```diff
@@ -612,6 +654,36 @@ function pmpro_rest_api_delete_membership_level( $request ) {
 			return new WP_REST_Response( $level->delete(), 200 );
 		}
 
+		/**
+		 * Get all membership levels.
+		 * @since TBD
+		 * Example: https://example.com/wp-json/pmpro/v1/membership_levels
+		 */
+		function pmpro_rest_api_get_membership_levels( $request ) {
+			
+			if ( ! class_exists( 'PMPro_Membership_Level' ) ) {
+				return new WP_REST_Response( 'Paid Memberships Pro level class not found.', 404 );
+			}
+
+			$params = $request->get_params();
+			$response_type = isset( $params['response_type'] ) ? sanitize_text_field( $params['response_type'] ) : null;
+
+			$levels = pmpro_getAllLevels( true, true );
+
+			// Hide confirmation message if not an admin or member.
+			if ( ! current_user_can( 'manage_options' ) && ! pmpro_hasMembershipLevel() ) {
```

Comment:
As coded this will show the confirmation for all levels to a user who has any 1 level.

Let's consider looping through and hiding the confirmations they don't have access to.

We also might need to allow users with the pmpro_membershiplevels capability to see the confirmation message. (i.e. membership managers or anyone else given that role)